website/content/en/status/report-2023-01-2023-03/daemon.adoc

```diff
@@ -0,0 +1,28 @@
+=== daemon(8) improvements
+
+Contact: Ihor Antonov <ihor@antonovs.family>
+Contact: Kyle Evans <kevans@FreeBSD.org>
+
+Links: +
+link:https://man.freebsd.org/cgi/man.cgi?daemon(8)[daemon]
+
+An ongoing effort to improve code quality and supervision capabilities of `daemon` utility.
+Daemon is a tool that can daemonize (send to background) or supervise any running process, automatically restart it if it crashes.
+Daemon is widely used in the ports tree and can be used more in base.
+
+This quarter `long_opts` support was added and the codebase went through initial refactoring phase to prepare it for further changes.
+There are no functional changes so far but more changes are coming.
+Please contact directly or on `#freebsd-dev` on libera if you encounter unexpected bugs.
+
+Planned work items for the next quarter:
+
+- use of kqueue for all event sources
+- fix link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=268580[Bug #268580]
+- fix link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=236117[Bug #236117]
+- fix link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=254511[Bug #254511]
+- fix link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=212829[Bug #212829]
+- `procctl PROC_REAP_ACQUIRE`
+
+We are looking forward for feedback, bug reports (old and new) and feature requests.
+
+
```

Comment:
Unnecessary blank lines at the end of file.

Reply:
fixed